packages/profiling-node/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@sentry/profiling-node",
+  "version": "1.3.5",
+  "description": "Official Sentry SDK for Node.js Profiling",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/profiling-node",
+  "author": "Sentry",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "types": "lib/types/index.d.ts",
+  "typesVersions": {
+    "<4.9": {
+      "lib/types/index.d.ts": [
+        "lib/types-ts3.8/index.d.ts"
+      ]
+    }
+  },
+  "bin": {
+    "sentry-prune-profiler-binaries": "scripts/prune-profiler-binaries.js"
+  },
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
Is this correct?

Reply:
Yeah, it will work, though we really want folks to be on 12+. I can change it, not sure anyone runs profiling under 14 anyways

Reply:
I think it's fine for now but once we drop support for Node <14 in the SDK we just bump to 14 here.